src/main/java/de/vogel612/testclient_javabot/TestProgram.java

```diff
@@ -61,10 +60,13 @@ public TestProgram(Stage stage) {
         new CoreBotCommands(chatInterface, bot).allCommands().forEach(bot::subscribe);
         LOGGER.info("Glued Core Commands");
 
+        // Used to shut down the executor when the window is closed
 		stage.setOnCloseRequest(new EventHandler<WindowEvent>() {
 			@Override
 			public void handle(WindowEvent event) {
+				LOGGER.info("Closing Window");
 				dispose();
+				LOGGER.info("Junior Client - Bye Bye!");
```

Comment:
It was the end :P
Anyways, just removing the "Closing Window" as I like Junior Client saying "Bye Bye" ;)